# Payment Capacity Assessment


The project aims to evaluate the creditworthiness of bank clients and provide valuable insights for the development of a credit scoring model.
The project analyzes the link between socio-economic factors of potential borrowers and their timely loan repayment ability.


**Data provided by the customer bank's credit department:**

- Number of children
- Marital status
- Income level
- Purpose of the loan

**RQs:**

1. Does the number of children impact timely loan repayment?
2. How does marital status affect loan repayment?
3. Is there a connection between income level and timely loan repayment?
4. How do loan purposes influence timely repayment?

**Tools and libraries:**

- Python, pandas

# Исследование надежности заемщиков

На данных статистики кредитного отдела банка-заказчика о платёжеспособности клиентов проводится исследование, результаты которого будут учтены при построении модели кредитного скоринга.

**Цель исследования** — оценить зависимость между возвратом кредита в срок и отдельными социально-экономическими параметрами потенциального кредитополучателя. Такими как: *количество детей, семейное положение, уровень дохода, цели кредита*.

**Ход исследования**
1. Обзор данных.
2. Предобработка данных.
3. Проверка гипотез.

## Обзор данных

In [1]:
# импортируем библиотеку pandas и считываем данные из csv-файла в датафрейм
import pandas as pd

try:
    data = pd.read_csv('/datasets/data.csv')
except:
    data = pd.read_csv('https://code.s3.yandex.net/datasets/data.csv')

In [2]:
# выведем первые 20 строчек датафрейма `data`
data.head(20)

children  days_employed  dob_years            education  education_id  \
0          1   -8437.673028         42               высшее             0   
1          1   -4024.803754         36              среднее             1   
2          0   -5623.422610         33              Среднее             1   
3          3   -4124.747207         32              среднее             1   
4          0  340266.072047         53              среднее             1   
5          0    -926.185831         27               высшее             0   
6          0   -2879.202052         43               высшее             0   
7          0    -152.779569         50              СРЕДНЕЕ             1   
8          2   -6929.865299         35               ВЫСШЕЕ             0   
9          0   -2188.756445         41              среднее             1   
10         2   -4171.483647         36               высшее             0   
11         0    -792.701887         40              среднее             1   
12         0            NaN         65              среднее             1   
13         0   -1846.641941         54  неоконченное высшее             2   
14         0   -1844.956182         56               высшее             0   
15         1    -972.364419         26              среднее             1   
16         0   -1719.934226         35              среднее             1   
17         0   -2369.999720         33               высшее             0   
18         0  400281.136913         53              среднее             1   
19         0  -10038.818549         48              СРЕДНЕЕ             1   

       family_status  family_status_id gender income_type  debt  \
0    женат / замужем                 0      F   сотрудник     0   
1    женат / замужем                 0      F   сотрудник     0   
2    женат / замужем                 0      M   сотрудник     0   
3    женат / замужем                 0      M   сотрудник     0   
4   гражданский брак                 1      F   пенсионер     0   
5   гражданский брак                 1      M   компаньон     0   
6    женат / замужем                 0      F   компаньон     0   
7    женат / замужем                 0      M   сотрудник     0   
8   гражданский брак                 1      F   сотрудник     0   
9    женат / замужем                 0      M   сотрудник     0   
10   женат / замужем                 0      M   компаньон     0   
11   женат / замужем                 0      F   сотрудник     0   
12  гражданский брак                 1      M   пенсионер     0   
13   женат / замужем                 0      F   сотрудник     0   
14  гражданский брак                 1      F   компаньон     1   
15   женат / замужем                 0      F   сотрудник     0   
16   женат / замужем                 0      F   сотрудник     0   
17  гражданский брак                 1      M   сотрудник     0   
18    вдовец / вдова                 2      F   пенсионер     0   
19         в разводе                 3      F   сотрудник     0   

     total_income                                 purpose  
0   253875.639453                           покупка жилья  
1   112080.014102                 приобретение автомобиля  
2   145885.952297                           покупка жилья  
3   267628.550329              дополнительное образование  
4   158616.077870                         сыграть свадьбу  
5   255763.565419                           покупка жилья  
6   240525.971920                       операции с жильем  
7   135823.934197                             образование  
8    95856.832424                   на проведение свадьбы  
9   144425.938277                 покупка жилья для семьи  
10  113943.491460                    покупка недвижимости  
11   77069.234271       покупка коммерческой недвижимости  
12            NaN                         сыграть свадьбу  
13  130458.228857                 приобретение автомобиля  
14  165127.911772              покупка жилой недвижимости  
15  116820.904450 

In [3]:
# выведем основную информацию о датафрейме с помощью метода info()
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 21525 entries, 0 to 21524
Data columns (total 12 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   children          21525 non-null  int64  
 1   days_employed     19351 non-null  float64
 2   dob_years         21525 non-null  int64  
 3   education         21525 non-null  object 
 4   education_id      21525 non-null  int64  
 5   family_status     21525 non-null  object 
 6   family_status_id  21525 non-null  int64  
 7   gender            21525 non-null  object 
 8   income_type       21525 non-null  object 
 9   debt              21525 non-null  int64  
 10  total_income      19351 non-null  float64
 11  purpose           21525 non-null  object 
dtypes: float64(2), int64(5), object(5)
memory usage: 2.0+ MB


## Предобработка данных

In [4]:
# выведем количество пропущенных значений для каждого столбца, используя комбинацию двух методов
data.isna().sum()

children               0
days_employed       2174
dob_years              0
education              0
education_id           0
family_status          0
family_status_id       0
gender                 0
income_type            0
debt                   0
total_income        2174
purpose                0
dtype: int64

В двух столбцах `days_employed` и `total_income` есть пропущенные значения. В `total_income` хранятся данные о доходах. 
На сумму дохода существенно влияет тип занятости, поэтому заполним пропуски в этом столбце медианным значением по каждому типу из столбца `income_type`.

Медианные значения представляют более объективную информация о доходах, в отличии от средних значений, которые могут некорректно характеризовать данные в случаях когда некоторые значения в датафрейме крайне выделяются среди большинства.

In [5]:
# перебираем каждый тип занятости в наборе уникальных значений столба income_type
for occupation in data['income_type'].unique():
    # на каждом шаге цикла с помощью атрибута loc выбираем строки,
    # в которых в income_type текущий тип занятости (occupation) и есть пропуски в total_income
    # и записываем в них медианное значение total_income строк с текущим типом занятости (occupation)
    data.loc[(data['income_type'] == occupation) & (data['total_income'].isna()), 'total_income'] = \
    data.loc[(data['income_type'] == occupation), 'total_income'].median()

# проверяем, что все пропуски заполнены
data['total_income'].isna().sum()

0

In [8]:
# обработаем аномальные значения в столбце days_employed
# заменим все отрицательные значения положительными с помощью метода abs()
data['days_employed'] = data['days_employed'].abs()

In [9]:
# определим медианное значение трудового стажа для каждого типа занятости
data.groupby('income_type')['days_employed'].median()

income_type
безработный        366413.652744
в декрете            3296.759962
госслужащий          2689.368353
компаньон            1547.382223
пенсионер          365213.306266
предприниматель       520.848083
сотрудник            1574.202821
студент               578.751554
Name: days_employed, dtype: float64

Два типа занятости (безработные и пенсионеры) представлены аномально большими значениями. Эти сведения не понадобятся для дальнейшего анализа, поэтому оставим значения в таком виде.

In [10]:
# применим метод unique() для поиска уникальных значений в столбце children
data['children'].unique()

array([ 1,  0,  3,  2, -1,  4, 20,  5])

В столбце `children` обнаружены два аномальных значения `-1` и `20`

In [11]:
# удалим строки методом логической индексации на основе двух условий
data = data[(data['children'] != -1) & (data['children'] != 20)]

In [12]:
# убедимся, что артефакты удалены
data['children'].unique()

array([1, 0, 3, 2, 4, 5])

In [13]:
# заполним пропуски в столбце days_employed медианными значениями по каждому типу занятости в income_type
# перебираем каждый тип занятости в наборе уникальных значений столба income_type
for occupation in data['income_type'].unique():
    # на каждом шаге цикла с помощью атрибута loc выбираем строки,
    # в которых в income_type текущий тип занятости (occupation) и есть пропуски в days_employed
    # и записываем в них медианное значение days_employed строк с текущим типом занятости (occupation)
    data.loc[(data['income_type'] == occupation) & (data['days_employed'].isna()), 'days_employed'] = \
    data.loc[(data['income_type'] == occupation), 'days_employed'].median()

In [14]:
# убедимся, что все пропуски заполнены
data.isna().sum()

children            0
days_employed       0
dob_years           0
education           0
education_id        0
family_status       0
family_status_id    0
gender              0
income_type         0
debt                0
total_income        0
purpose             0
dtype: int64

In [15]:
# заменим вещественный тип данных в столбце total_income на целочисленный
# используем метод astype() с аргументом ('int') для перевода типа значения в целое число
data['total_income'] = data['total_income'].astype(int)

In [16]:
# обработаем неявные дубликаты в столбце education
# методом lower() приведём все символы в столбцах education к нижнему регистру
data['education'] = data['education'].str.lower()

In [17]:
# посчитаем количество строк-дубликатов в столбце с помощью двух методов
data.duplicated().sum()

71

In [18]:
# удалим дубликаты
data = data.drop_duplicates()

In [19]:
# методом lower() приведём все символы в столбцах amily_status к нижнему регистру
data['family_status'] = data['family_status'].str.lower()

Создадим собственную функцию, которая назначает клиенту категорию в зависимости от ежемесячного дохода на основании диапазонов:
- 0–30000 — `'E'`;
- 30001–50000 — `'D'`;
- 50001–200000 — `'C'`;
- 200001–1000000 — `'B'`;
- 1000001 и выше — `'A'`.

In [20]:
# создадим функцию categorize_income(), используя конструкцию try - except для обработки исключений
def categorize_income(income):
    try:
        if income <= 30000:
            return 'E'
        elif income <= 50000:
            return 'D'
        elif income <= 200000:
            return 'C'
        elif income <= 1000000:
            return 'B'
        elif income >= 1000001:
            return 'A'
    except:
        pass

In [21]:
# создадим отдельный столбец с категориями клиентов, 
# в ячейках столбца будут записаны возвращаемые функцией значения
# применим функцию методом apply()
data['total_income_category'] = data['total_income'].apply(categorize_income)

In [22]:
# используем метод unique() для поиска уникальных значений в столбце purpose
data['purpose'].unique()

array(['покупка жилья', 'приобретение автомобиля',
       'дополнительное образование', 'сыграть свадьбу',
       'операции с жильем', 'образование', 'на проведение свадьбы',
       'покупка жилья для семьи', 'покупка недвижимости',
       'покупка коммерческой недвижимости', 'покупка жилой недвижимости',
       'строительство собственной недвижимости', 'недвижимость',
       'строительство недвижимости', 'на покупку подержанного автомобиля',
       'на покупку своего автомобиля',
       'операции с коммерческой недвижимостью',
       'строительство жилой недвижимости', 'жилье',
       'операции со своей недвижимостью', 'автомобили',
       'заняться образованием', 'сделка с подержанным автомобилем',
       'получение образования', 'автомобиль', 'свадьба',
       'получение дополнительного образования', 'покупка своего жилья',
       'операции с недвижимостью', 'получение высшего образования',
       'свой автомобиль', 'сделка с автомобилем',
       'профильное образование', 'высшее об

Создадим собственную функцию, которая категоризует кредитные операции в соответствии с данными в столбце `purpose`.
Новый столбец `purpose_category` будет хранить следующие категории:
- `'операции с автомобилем'`,
- `'операции с недвижимостью'`,
- `'проведение свадьбы'`,
- `'получение образования'`.

In [23]:
# создадим функцию categorize_purpose(), используя конструкцию try - except для обработки исключений
def categorize_purpose(row):
    try:
        if 'автом' in row:
            return 'операции с автомобилем'
        elif 'жил'in row or 'недвиж' in row:
            return 'операции с недвижимостью'
        elif 'свадьб' in row:
            return 'проведение свадьбы'
        elif 'образов' in row:
            return 'получение образования'
    except:
        return 'нет категории'

In [24]:
# создадим отдельный столбец с категориями кредитных операций, 
# в ячейках столбца будут записаны значения, возвращаемые функцией
# применим функцию методом apply()
data['purpose_category'] = data['purpose'].apply(categorize_purpose)
data.head()

children  days_employed  dob_years education  education_id  \
0         1    8437.673028         42    высшее             0   
1         1    4024.803754         36   среднее             1   
2         0    5623.422610         33   среднее             1   
3         3    4124.747207         32   среднее             1   
4         0  340266.072047         53   среднее             1   

      family_status  family_status_id gender income_type  debt  total_income  \
0   женат / замужем                 0      F   сотрудник     0        253875   
1   женат / замужем                 0      F   сотрудник     0        112080   
2   женат / замужем                 0      M   сотрудник     0        145885   
3   женат / замужем                 0      M   сотрудник     0        267628   
4  гражданский брак                 1      F   пенсионер     0        158616   

                      purpose total_income_category          purpose_category  
0               покупка жилья                     B  операции с недвижимостью  
1     приобретение автомобиля                     C    операции с автомобилем  
2               покупка жилья                     C  операции с недвижимостью  
3  дополнительное образование                     B     получение образования  
4             сыграть свадьбу                     C        проведение свадьбы

## Исследование данных и проверка гипотез

На предварительном этапе мы изучили основную информацию о датафрейме, определили количество и тип данных в таблице, обработали аномальные значения и пропуски, избавились от дубликатов и убедились, что все данные отображаются корректно.

Кроме того, мы создали функции `categorize_income()` и `categorize_purpose()`, которые позволили объединить данные отдельных столбцов в категории, чтобы численности каждой группы было достаточно для формулировки выводов.

В таком виде данные готовы для более подробного исследовательского анализа, чтобы выяснить существует ли зависимость между возвратом кредита в срок и отдельными социально-экономическими параметрами потенциального кредитополучателя. Такими как:
- количество детей,
- семейное положение,
- уровень дохода,
- цели кредита.

Отметим, что датафрейм не представляет содержательных сведений о кредитной истории клиента, а ограничевается фактом наличия/отсутствия задолженности по возврату кредитов (`debt`). На наш взгляд, наличие более полных данных (например, сколько займов выдали и в скольких случаях отказали, на какой срок были задержки по выплате) могло бы существенно увеличить точность прогноза в отношении вероятности возврата кредита в срок.

В этой связи, предсказание кредитных рисков в данном исследовании будет исходить из наблюдаемых зависимостей между указанными социально-экономическими параметрами. Это позволит оценить существующую финансовую нагрузку клиента и сформулировать вероятность возврата кредита в срок.

**Есть ли зависимость между количеством детей и возвратом кредита в срок?**

In [25]:
# посчитаем значения в столбце children методом value_counts()
# узнаем сколько строк приходится на каждую группу
data['children'].value_counts()

0    14091
1     4808
2     2052
3      330
4       41
5        9
Name: children, dtype: int64

- Количество строк в группах сильно различается. Это говорит о том, что выборки не сбалансированы.
- Очевидно, что самый многочисленный сегмент образуют семьи без детей. 
- Среди клиентов с детьми наиболее многочисленный сегмент составляют семьи с одним ребенком.
- В целом, количество потенциальных заёмщиков снижается с увеличением количества детей в семье. Клиенты с тремя и более детьми в семье встречаются в выборке довольно редко. В этой связи, для обеспечения достоверности результатов анализа мы оставим только сопоставимые по размеру группы.

In [26]:
# при помощи логической индексации удалим строки на основе соответствующих условий
child_group_balance = data[(data['children'] != 3) & (data['children'] != 4) & (data['children'] != 5)]

In [27]:
# определим зависимость между показателем задолженности по возврату кредитов и наличием детей в семье
# 1) группируем данные по количеству детей в семье (children) с помощью groupby('children')
# 2) достаём из сгруппированной таблицы только столбец с данными о задолженности по возврату кредитов ('debt')
# 3) рассчитываем процент погашения кредита по каждой группе с количеством детей
child_loan_repayment = child_group_balance.groupby('children')['debt'].mean() * 100
child_loan_repayment

children
0    7.543822
1    9.234609
2    9.454191
Name: debt, dtype: float64

In [28]:
# создадим сводную таблицу для набора data_balance с помощью метода pivot_table()
child_data_pivot = child_group_balance.pivot_table(index='children', values='debt', aggfunc={'debt': ['count', 'mean']})
# создадим в таблице child_data_pivot новые столбцы и сохраним в них значения по заёмщикам и просрочкам
child_data_pivot.columns = ['borrowers', 'payment_delay']
# рассчитаем процент по каждой группе
child_data_pivot['payment_delay'] = child_data_pivot['payment_delay'] * 100
# отсортируем таблицу по столбцу 'payment_delay' в порядке возрастания
child_data_pivot.sort_values(by='payment_delay')

borrowers  payment_delay
children                          
0           14091.0       7.543822
1            4808.0       9.234609
2            2052.0       9.454191

**Вывод:** 
- Вероятность возврата кредита в срок выше среди потенциальных заёмщиков без детей. Отмечается разница по возврату кредитов заёмщиками без детей и при наличии детей в семье. Вероятно наличие детей в семье обуславливает дополнительную финансовую нагрузку, что в свою очередь может стать причиной просрочек погашения кредита.
- Количество детей в семье хотя и не существенно, но влияет на показатель возврата кредита в срок. Доля просрочек в семьях с двумя детьми возрастает по сравнению с семьями с одним ребёнком. Вместе с тем, такие выводы нуждаются в уточнении при учёте дополнительных сведений о составе семьи (например, один родитель воспитывающий двоих детей и др.).

**Есть ли зависимость между семейным положением и возвратом кредита в срок?**

In [29]:
# посчитаем значения в столбце family_status методом value_counts()
data['family_status'].value_counts()

женат / замужем          12261
гражданский брак          4134
не женат / не замужем     2796
в разводе                 1189
вдовец / вдова             951
Name: family_status, dtype: int64

In [30]:
# определим зависимость между показателем задолженности по возврату кредитов и семейным положением
# 1) группируем данные по столбцу (family_status) с помощью метода groupby()
# 2) достаём из сгруппированной таблицы только столбец с данными о задолженности по возврату кредитов ('debt')
# 3) рассчитываем процент погашения кредита по каждой группе
marital_status_group = data.groupby('family_status')['debt'].mean() * 100
marital_status_group

family_status
в разводе                7.064760
вдовец / вдова           6.624606
гражданский брак         9.313014
женат / замужем          7.560558
не женат / не замужем    9.763948
Name: debt, dtype: float64

In [28]:
# создадим сводную таблицу по результатам с помощью метода pivot_table()
marital_status_data_pivot = data.pivot_table(index='family_status', values='debt', aggfunc={'debt': ['count', 'mean']})
# создадим в таблице marital_status_data_pivot новые столбцы и сохраним в них значения по заёмщикам и просрочкам
marital_status_data_pivot.columns = ['borrowers', 'payment_delay']
# рассчитаем процент по каждой группе
marital_status_data_pivot['payment_delay'] = marital_status_data_pivot['payment_delay'] * 100
# отсортируем таблицу по столбцу 'payment_delay' в порядке возрастания
marital_status_data_pivot.sort_values(by='payment_delay')

borrowers  payment_delay
family_status                                  
вдовец / вдова             951.0       6.624606
в разводе                 1189.0       7.064760
женат / замужем          12261.0       7.560558
гражданский брак          4134.0       9.313014
не женат / не замужем     2796.0       9.763948

**Вывод:** 
- Наименьшая вероятность задолженности по возврату кредитов наблюдается у потенциальных заёмщиков в статусе `вдовец / вдова`. Возможно это связано с получением государственной помощи в виде выплат и компенсаций.
- Наиболее высокий показатель просроченных платежей по сравнению с другими группами представляют клиенты в статусе `не женат / не замужем`. Это может быть связано с отсутствием финансовой поддержки со стороны партнера, а также менее зависимым поведением в плане организации бюджета.

**Есть ли зависимость между уровнем дохода и возвратом кредита в срок?**

In [31]:
# оценим значения в столбце total_income_category методом value_counts()
data['total_income_category'].value_counts()

C    15921
B     5014
D      349
A       25
E       22
Name: total_income_category, dtype: int64

Количество строк в группах существенно различается. Это говорит о том, что выборки не сбалансированы. В этой связи, для обеспечения достоверности результатов анализа оставим наиболее сопоставимые по размеру категории.

In [32]:
# при помощи логической индексации удалим строки на основе соответствующих условий
income_group_balance = data[(data['total_income_category'] != 'E') & (data['total_income_category'] != 'A') & (data['total_income_category'] != 'D')]

In [33]:
# определим зависимость между показателем задолженности по возврату кредитов и уровнем дохода
# 1) группируем данные по столбцу (total_income_category) с помощью метода groupby()
# 2) достаём из сгруппированной таблицы только столбец с данными о задолженности по возврату кредитов ('debt')
# 3) рассчитываем процент погашения кредита по каждой группе
income_level_group = income_group_balance.groupby('total_income_category')['debt'].mean() * 100
income_level_group

total_income_category
B    7.060231
C    8.498210
Name: debt, dtype: float64

In [34]:
# создадим сводную таблицу для набора income_group_balance с помощью метода pivot_table()
income_data_pivot = income_group_balance.pivot_table(index='total_income_category', values='debt', aggfunc={'debt': ['count', 'mean']})
# создадим в таблице income_data_pivot новые столбцы и сохраним в них значения по заёмщикам и просрочкам
income_data_pivot.columns = ['borrowers', 'payment_delay']
# рассчитаем процент по каждой группе
income_data_pivot['payment_delay'] = income_data_pivot['payment_delay'] * 100
# отсортируем таблицу по столбцу 'payment_delay' в порядке возрастания
income_data_pivot.sort_values(by='payment_delay')

borrowers  payment_delay
total_income_category                          
B                         5014.0       7.060231
C                        15921.0       8.498210

**Вывод:** 
Категория клиентов `B` с доходом от 200 000 до 1 000 000 с бóльшей долей вероятности возвращает кредит в срок по сравнению с заёмщиками из категории `C` с доходом от 50 000 до 200 000 рублей в месяц. Это вполне согласуется с представлениями о платежеспособности потенциальных клиентов при оценке кредитных рисков.

**Как разные цели кредита влияют на его возврат в срок?**

In [35]:
# оценим значения в столбце purpose_category методом value_counts()
data['purpose_category'].value_counts()

операции с недвижимостью    10751
операции с автомобилем       4279
получение образования        3988
проведение свадьбы           2313
Name: purpose_category, dtype: int64

In [36]:
# оценим зависимость между показателем задолженности по возврату кредитов и цели получения кредита
# 1) группируем данные по столбцу (purpose_category) с помощью метода groupby()
# 2) достаём из сгруппированной таблицы только столбец с данными о задолженности по возврату кредитов ('debt')
# 3) рассчитываем процент погашения кредита по каждой группе
purpose_loan_group = data.groupby('purpose_category')['debt'].mean() * 100
purpose_loan_group

purpose_category
операции с автомобилем      9.347978
операции с недвижимостью    7.255139
получение образования       9.252758
проведение свадьбы          7.911803
Name: debt, dtype: float64

In [148]:
# создадим сводную таблицу по результатам с помощью метода pivot_table()
purpose_data_pivot = data.pivot_table(index='purpose_category', values='debt', aggfunc={'debt': ['count', 'mean']})
# создадим в таблице purpose_data_pivot новые столбцы и сохраним в них значения по заёмщикам и просрочкам
purpose_data_pivot.columns = ['borrowers', 'payment_delay']
# рассчитаем процент по каждой группе
purpose_data_pivot['payment_delay'] = purpose_data_pivot['payment_delay'] * 100
# отсортируем таблицу по столбцу 'payment_delay' в порядке возрастания
purpose_data_pivot.sort_values(by='payment_delay')

borrowers  payment_delay
purpose_category                                  
операции с недвижимостью    10751.0       7.255139
проведение свадьбы           2313.0       7.911803
получение образования        3988.0       9.252758
операции с автомобилем       4279.0       9.347978

**Вывод:** 
- Клиенты с целью получения кредита на `недвижимость и жилье` с бóльшей вероятностью осуществляют платежи в срок. 
- Тогда как займы связанные с `автомобилем` или `получением образования` имеют наибольшую долю просрочек.
- Такие показатели могут свидетельствовать о том, что риск потерять жилье стимулирует заёмщиков вносить кредитные платежи в срок.

**Общий вывод**

Предсказание кредитных рисков в данном исследовании основано на наблюдении зависимостей между социально-экономическими параметрами, что позволило оценить существующую финансовую нагрузку клиента и сформулировать вероятность возврата кредита в срок.
- **Наличие детей в семье** связано с бóльшим количеством задолженностей по возврату кредитов (9.45%) по сравнению с заёмщиками без детей (7.54%). Очевидно, наличие детей в семье обуславливает дополнительную финансовую нагрузку, что в свою очередь может стать причиной просрочек по погашению кредита.
- **Семейный статус** не женат / не замужем связан с более высоким показателем просроченных платежей по сравнению с другими группами (9.76%). Это может быть связано с отсутствием финансовой поддержки со стороны партнера, а также менее зависимым поведением в плане организации бюджета. Наименьшая вероятность просрочек наблюдается у заёмщиков в статусе вдовец / вдова (6.62%). Возможно это связано с получением государственной помощи в виде выплат и компенсаций.
- **Уровень дохода** от 200 000 до 1 000 000 рублей в месяц связан с наименьшим показателем просроченных платежей: 7.06% против 8.50% в категории от 50 000 до 200 000 рублей в месяц, что вполне согласуется с представлениями о платежеспособности потенциальных клиентов при оценке кредитных рисков.
- Клиенты с **целью получения кредита** на недвижимость и жилье с бóльшей вероятностью возвращают платежи в срок (7.26%). Тогда как займы связанные с автомобилем или получением образования имеют наибольшую долю просрочек (9.35% и 9.25%). Такие показатели могут свидетельствовать о том, что риск потерять жилье стимулирует заёмщиков вносить кредитные платежи в срок.

Наблюдаемые различия в отношении зависимости между возвратом кредита в срок и отдельными социально-экономическими параметрами потенциального заёмщика говорят о том, что учёт таких данных обоснован при построении модели кредитного скоринга и оценки платёжеспособности клиентов.

На наш взгляд, наличие более полных данных о кредитной истории потенциального заёмщика (например, сколько займов выдали и в скольких случаях отказали, на какой срок были задержки по выплате) могло бы существенно увеличить точность прогноза в отношении вероятности возврата кредита в срок.